app/src/main/java/com/nutomic/syncthingandroid/views/FoldersAdapter.java

```diff
@@ -36,7 +36,7 @@
 
     private static final String TAG = "FoldersAdapter";
 
-    private final HashMap<String, Model> mModels = new HashMap<>();
+    private final HashMap<String, FolderStatus> mFolderStatus = new HashMap<>();
```

Comment:
Statuses

Reply:
I assume this is local stuff and that double map stuff is remote? Its not obvious from variable names

Reply:
Renamed to mLocalFolderStatuses